server/test/user.js

```diff
@@ -14,11 +14,14 @@ const { forceSyncDB, User, Profile, Map, MapInfo, Activity, MapCredit, MapStats,
 
 chai.use(chaiHttp);
 
+let fs = require('fs');
```

Comment:
Put all `require` statements up top. Put this below `activity = require ( ... );`